tests/test_simpleAssociation.py

```diff
@@ -0,0 +1,226 @@
+# This file is part of pipe_tasks.
+
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+import numpy as np
+import pandas as pd
+import unittest
+
+import lsst.afw.table as afwTable
+import lsst.geom as geom
+import lsst.utils.tests
+from lsst.pipe.tasks.associationUtils import query_disc, eq2xyz, toIndex
+from lsst.pipe.tasks.simpleAssociation import (SimpleAssociationTask,
+                                               SimpleAssociationConfig)
+
+class TestSimpleAssociation(lsst.utils.tests.TestCase):
+
+    def setUp(self):
+        simpleAssoc = SimpleAssociationTask()
+        self.tractPatchId = 1234
+        self.skymapBits = 16
+
+        self.nDiaObjects = 10
+        self.diaObjRas = np.linspace(45, 46, self.nDiaObjects)
+        self.diaObjDecs = np.linspace(45, 46, self.nDiaObjects)
+        # Copy a coord to get multiple matches.
+        self.diaObjRas[3] = self.diaObjRas[2] + 0.1 / 3600
+        self.diaObjDecs[3] = self.diaObjDecs[2] + 0.1 / 3600
+        self.diaObjects = [
+            simpleAssoc.createDiaObject(objId, ra, decl)
+            for objId, ra, decl in zip(
+                np.arange(self.nDiaObjects, dtype=int),
+                self.diaObjRas,
+                self.diaObjDecs)]
+
+        self.hpIndices = [toIndex(simpleAssoc.config.nside,
+                                  diaObj["ra"],
+                                  diaObj["decl"])
```

Comment:
Why do you use a mixture of decl and dec as abbreviations?

Reply:
decl is the column name in the DPDD and our Apdb schema, hence use of it.